tests/test_indexer.py

```diff
@@ -87,6 +88,8 @@ def tearDownModule():
 class TestIndexerBase(ElasticsearchTestCase, DSSAssertMixin, DSSStorageMixin, DSSUploadMixin, metaclass=ABCMeta):
     bundle_key_by_replica = dict()  # type: typing.MutableMapping[str, str]
 
+    backends = [ElasticsearchIndexBackend]
```

Comment:
nit: here again, might be good to have this part of a global config so everything is synced

Reply:
Part of the benefit of the separation of the indexing logic and the actual storage implementation is that this could be replaced with an in-memory implementation, no? z.B. have a class InMemoryIndexBackend(IndexBackend). This might finally make this test fast!

If possible, I guess this improvement could go in a different commit.

Reply:
Fixed the nit.

Fast yes, but also anemic. The meat of the indexer is still in the ES backend, and is ES-specific. If we cut that out, there wouldn't be much left to test. Another cause of slowness is the fact that the metadata is still read from the buckets everytime for every test. That functionality should be tested once, and mocked for the rest of the tests.